content/tutorials/xrp-ledger-businesses/list-your-exchange-on-xrp-charts.md

```diff
@@ -0,0 +1,186 @@
+# List Your Exchange on XRP Charts
+
+In addition to providing data about the XRP Ledger network and its accounts and transactions, XRP Charts also provides [XRP market data](https://xrpcharts.ripple.com/#/xrp-markets). ***TODO: Correct? Is this the area of the site that exchanges are asking to be listed in? Live Chart, Multi Markets, Active Accounts, XRP Markets?*** This tutorial describes how to have your exchange and its XRP trade and order book data listed on XRP Charts.
+
+To enable XRP Charts to list your exchange, complete the following tasks:
+
+1. Provide a [Get Recently Executed Trades](#provide-a-get-recently-executed-trades-endpoint) REST API endpoint.
+
+2. Provide a [Get Current Order Book](#provide-a-get-current-order-book-endpoint) REST API endpoint.
+
+3. [Send a listing request to XRP Charts](#send-a-listing-request-to-xrp-charts).
+
+If you have any questions about endpoint specifications, contact xxx@xxxxx.com. ***TODO: do we want to provide support? This will help avoid them guessing while building and then find out that their endpoint doesn't meet our specs only after they've requested listing.***
+
+
+## Provide a Get Recently Executed Trades Endpoint
+
+***TODO: Do we really mean "recently" only? If yes, how recent? last 500-1000 as described below?***
+
+Provide a REST API endpoint that returns data about individual trades executed in a particular market. ***TODO: is this about trades that involve XRP as a base or counter curr only? I see a wider variety of currencies listed on the site.***
+
+XRP Charts will query this endpoint at least every 15-30 seconds. The endpoint must be capable of returning 500-1,000 trades per response, but that number could be smaller when the `last_tid` query parameter is included. ***TODO: What is the purpose of this spec, just so I can better explain it? 500-1000 trades is a response without any query parameters. Does this mean that the endpoint doesn't have to be able to return anything older than the last 1,000 trades (since we specifically call this "get recently executed trades")? Or does the endpoint need to be able to respond to a request with a tid from 1 year ago - returning every trade since that tid - which could be more than 1000? Or is this saying that the endpoint needs to provide pagination that enables XRP Charts to request results in sets of 500 to 1,000 at a time -- with a query parameter of a tid from last year, for example?***
+
+
+### Request Format
+
+Provide a request format like the following:
+
+```
+GET {api_base_url}/v1/trades
+```
+
+***TODO: any authentication specs? or publicly accessible?***
+
+#### Query Parameters
+
+Provide the following query parameters. XRP Charts will optionally use them to narrow the trades returned in the response. ***TODO: true? will XRP Charts always make this call with both query params present? Or also with just one or the other, or none?***
```

Comment:
They don't specifically have to be query params, nor follow this naming convention, and the last_tid is not required, but if its available I'll use it.  Another potentially useful param would be a `limit` or `count`